test/cpp/tensorexpr/test_kernel.cpp

```diff
@@ -1969,10 +1969,7 @@ TEST_F(Kernel, FuseLoopsWithVariableConcatDim) {
   # CHECK: for (int64_t i
   # CHECK-NEXT: for (int64_t j
   # CHECK-NEXT: for (int64_t k
-  # CHECK: for (int64_t j
-  # CHECK-NEXT: for (int64_t k
-  # CHECK: for (int64_t j
-  # CHECK-NEXT: for (int64_t k
+  # CHECK-NOT: for (int64_t j
```

Comment:
We started to fuse this case, right? Can we also now add a test for a case which we shouldn't be fusing?

Reply:
Sure, added another test for that.